src/serialization/tokenizer.hpp

```diff
@@ -141,7 +141,7 @@ class tokenizer
 
 	/**
 	 * increments the line number if the current character is a newline
-	 * set current_ to the next character that's not \r
+	 * set current_ to the next character that's not \\r
```

Comment:
Why are you doubling the backslashes here? That doesn't make any sense.

Reply:
Otherwise doxygen fails to parse it since it thinks it's an invalid doxygen command.

Reply:
If that's the reason, I'd say the correct fix is instead to surround it with backticks, ie `\r` and `\r\n`. Since Doxygen parses Markdown, hopefully that'll be sufficient.